src/main/java/school/hei/haapi/endpoint/rest/mapper/DelatyPenaltyMapper.java

```diff
@@ -0,0 +1,27 @@
+package school.hei.haapi.endpoint.rest.mapper;
+
+import org.springframework.stereotype.Component;
+import school.hei.haapi.endpoint.rest.model.DelayPenalty;
+import school.hei.haapi.model.Group;
+
+@Component
+public class DelatyPenaltyMapper {
+  public DelayPenalty toRest (school.hei.haapi.model.DelayPenalty domain){
+    return new DelayPenalty()
+        .id(domain.getId())
+        .interestPercent(domain.getInterestPercent())
+        .interestTimerate(domain.getInterestTimeRate())
+        .graceDelay(domain.getGraceDelay())
+        .applicabilityDelayAfterGrace(domain.getApplicabilityDelayAfterGrace())
+        .creationDatetime(domain.getCreationDatetime());
+  }
+  public DelayPenalty toDomain(school.hei.haapi.model.DelayPenalty toUpdate) {
```

Comment:
In toDomain, expect ..model.DelayPenalty and as parameter we have ...rest.model.DelayPenalty